autodeconstruct.lua

```diff
@@ -137,24 +136,52 @@ function autodeconstruct.on_built_entity(event)
     end
 end
 
+function autodeconstruct.build_pipes(drill)
+    local x = drill.position.x
+    local y = drill.position.y
+    local dir = drill.direction
+    -- create pipes for each fluid connector (ignore the side with the ore output)
+    game.surfaces[1].create_entity{name="entity-ghost", position = {x = x, y = y}, force=game.players[1].force, inner_name="pipe"}
```

Comment:
Could use the surface where the drill was, and the drills last users force to make this a more general solution.